src/isar_exr/api/graphql_client.py

```diff
@@ -35,7 +35,7 @@ def _initialize_client(self):
         transport = AIOHTTPTransport(url=settings.ROBOT_API_URL, headers=auth_header)
         self.client = Client(transport=transport, fetch_schema_from_transport=True)
 
-    def query(self, json_body: str) -> Dict[str, Any]:
+    def query(self, gql_query: DocumentNode, query_parameters: dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I think we should leave this as a str to fully abstract the graph ql client from the rest of the codebase